pg_back

```diff
@@ -282,6 +293,10 @@ do
     else
         dumped+=( "$db" )
     fi
+    if [ -n "${SIGNATURE_ALGO}" -a "${SIGNATURE_ALGO}" != "none" ]; then
+        info "creating signature file for $(basename ${dump}) into $(basename ${dump}).${SIGNATURE_ALGO}"
+        ${SIGNATURE_ALGO}sum ${dump} | awk '{ print $1 }' > ${dump}.${SIGNATURE_ALGO}
```

Comment:
Why removing the filename? It prevents a future use of XXXsum -c to verify the checksum.

Reply:
Hi Nicolas(@orgrim),

We want clean up signatures at same time we purge our dump files after 30 days by default.

So it's very ugly to get signature files without any related dump file.

Reply:
Hi,
There is a misunderstanding, I was taking about the pipe to awk that removes the filename in the output of sha1sum (and friends). Removing it prevents the use of sha1sum -c to check the signature.

Regards,

Reply:
Last PR solves this point by removing awk and pipe.